### PR TITLE
Changed compile to implementation and testCompile to testImplementation

### PR DESCRIPTION
**Pull request by evandrojr**
DONE:

compile -> implementation
testCompile -> testImplementation

To do:
debugCompile -> debugImplementation
androidTestCompile -> androidTestImplementation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
 # maven2gradle
 converts maven dependencies to gradle format
 
-**check it out in [here](http://sagioto.github.io/maven2gradle/)**
+**check it out in [here](https://evandrojr.github.io/maven2gradle/)**
```
Comment (IRus) [inline README.md]: You should leave original repo path

File: index.html [hunk 2]
```diff
@@ -8,7 +8,7 @@
     <script src="converter.js"></script>
 </head>
 <body>
-<a href="https://github.com/sagioto/maven2gradle"><img style="position: absolute; top: 0; right: 0; border: 0;" src="https://camo.githubusercontent.com/38ef81f8aca64bb9a64448d0d70f1308ef5341ab/68747470733a2f2f73332e616d617a6f6e6177732e636f6d2f6769746875622f726962626f6e732f666f726b6d655f72696768745f6461726b626c75655f3132313632312e706e67" alt="Fork me on GitHub" data-canonical-src="https://s3.amazonaws.com/github/ribbons/forkme_right_darkblue_121621.png"></a>
+<a href="https://github.com/evandrojr/maven2gradle"><img style="position: absolute; top: 0; right: 0; border: 0;" src="https://camo.githubusercontent.com/38ef81f8aca64bb9a64448d0d70f1308ef5341ab/68747470733a2f2f73332e616d617a6f6e6177732e636f6d2f6769746875622f726962626f6e732f666f726b6d655f72696768745f6461726b626c75655f3132313632312e706e67" alt="Fork me on GitHub" data-canonical-src="https://s3.amazonaws.com/github/ribbons/forkme_right_darkblue_121621.png"></a>
```
Comment (IRus) [inline index.html]: You should leave original repo path

File: README.md (hunk 1 above)
Comment (evandrojr) [reply README.md]: Sorry it was not supposed to go in the PR

**Review by evandrojr — COMMENTED** [review]
Using original repo address

**Comment by IRus** [comment]
@sagioto do you mind merge this change set?

**Comment by evandrojr** [comment]
Do you mean in my fork? I can do It. No problem.

**Comment by IRus** [comment]
@evandrojr no, I'm kindly asking @sagioto to merge this PR :)

**Comment by evandrojr** [comment]
> @evandrojr no, I'm kindly asking @sagioto to merge this PR :)

Got it! I was in a hurry when I read the message.

**Review by sagioto — APPROVED** [review]

**Pull request merged**